src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall(Lottery ball) {
+        int number = new Random().nextInt(100);
```

Comment:
Magic number `100` should be extracted to a constant field with a meaningful name to improve code readability and maintainability.